.bazelrc

```diff
@@ -40,6 +40,7 @@ build --java_language_version=8
 # Override PGV validation with NOP functions
 build --@com_envoyproxy_protoc_gen_validate//bazel:template-flavor=nop
 
+build:dbg --compilation_mode=debug
```

Comment:
isn't it `--compilation_mode=dbg`?

Reply:
(I think that for whatever reason both work since the change from debug -> dbg did not result in a rebuild of the project)

Reply:
Mhm... looking into this.

Reply:
Yea that's because android studio's bazel plugin is adding `-c dbg` when you hit the lil debug icon and is override your compilation mode, so I presume you can set any garbage and it won't fail because it'll get overriden

Reply:
ah, got it - makes sense. Thank you for the context.

In that case, I am going to keep `--compilation_mode=debug` for cases when we want to use `--config=dbg` from outside of the debugging flow in Android Studio UI.